scripts/check_docs.py

```
@@ -0,0 +1,275 @@
+#!/usr/bin/env python
```

Ruff (0.13.2)

Shebang is present but file is not executable

(EXE001)